docs/multicluster/user-guide.md

```diff
@@ -424,72 +438,64 @@ spec:
   serviceCIDR: 110.96.0.0/20
 ```
 
-Make sure you repeat the same steps to assign a Gateway Node in all member
-clusters, then `antrea-agent` will set up Geneve tunnels among the Gateway Nodes
-of member clusters based on the local `Gateway` and the `ClusterInfoImport`
-resources, and route Multi-cluster Service traffic across clusters through the
-tunnels. `antrea-agent` on regular Nodes will route cross-cluster traffic from
-local Pods to the Gateway Node of the member cluster.
-
-Once you confirm that all `Gateway` and `ClusterInfoImport` resources are
+Make sure you repeat the same step to assign a Gateway Node in all member
+clusters. Once you confirm that all `Gateway` and `ClusterInfoImport` are
 created correctly, you can follow the [Multi-cluster Service](#multi-cluster-service)
-section to create Multi-cluster Services and verify cross-cluster Service
+section to create multi-cluster Services and verify cross-cluster Service
 access.
 
 ## Multi-cluster Service
 
-After you set up a ClusterSet properly, you can simply create a `ServiceExport` resource
-as below to export a `Service` from one member cluster to other members in the ClusterSet,
-you can update the name and Namespace according to your local K8s Service.
+After you set up a ClusterSet properly, you can create a `ServiceExport` CR to
+export a Service from one cluster to other clusters in the Clusterset, like the
+example below:
 
 ```yaml
 apiVersion: multicluster.x-k8s.io/v1alpha1
 kind: ServiceExport
 metadata:
   name: nginx
-  namespace: kube-system
+  namespace: default
 ```
 
-For example, once you export the `kube-system/nginx` Service in the member cluster
-`test-cluster-west`, Antrea Multi-cluster Controller in the member cluster will create
-two corresponding `ResourceExport` resources in the leader cluster, and the controller
-in leader cluster will create two `ResourceImport` contains all exported Service and
-Endpoints'. you can check the created resources in the leader cluster which should be
-like below:
+For example, once you export the `default/nginx` Service in member cluster
```

Comment:
Revised this section to remove "Multi-cluster Controller", etc. and some internal behaviors, as no need to talk about design in user guide.